CoordgenTemplates.h

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Does this file need to be added to the `install` list in CMakeLists.txt?
It looks like all the other header files are there, but I'm not sure all of them are actually needed (that'd be another PR to fix).

Reply:
I totally didn't even think about that.  But I don't think so, because the function that CoordgenTemplates.h has hidden visibility, so a client can't use the function.